templates/auth/signup.html

```diff
@@ -35,7 +35,7 @@ <h5>Register</h5>
         </div>
         {% endif %}
         <div class="grid-x">
-            <div class="medium-12 columns">
+            <div class="medium-6 columns">
```

Comment:
Why modify this, but not the rest?

Reply:
Because if we change the others then login button and forget you password will come on same line.